tests/functional/a/assignment_expression.py

```diff
@@ -49,6 +52,24 @@ def func():
 print(function())
 
 
+# https://github.com/PyCQA/pylint/issues/3763
```

Comment:
Nitpick : We can put new test case at the end of files to have a smaller diffs :)

Reply:
Of course we could, but is it really worth it for 7 lines more? With the current order, it at least makes somewhat sense.